.changes/v3.0.0/588-removals.md

```diff
@@ -0,0 +1,5 @@
+* Removed deprecated attributes `external_networks, default_gateway_network` from `vcd_edgegateway` [GH-588]
+* Removed `vcd_independent_disk.size` in favor for `vcd_independent_disk.size_in_mb` [GH-588]
+* Removed deprecated attributes `template_name, catalog_name, network_name, memory, cpus, ip, storage_profile, initscript, ovf, accept_all_eulas, power_on` from `vcd_vapp` [GH-588]
```

Comment:
1. Missing a line for `vcd_network` removal.
2. Please put all this information into docs (same ask as https://github.com/vmware/terraform-provider-vcd/pull/563#discussion_r497367021 ).

Reply:
vcd_network removed by @dataclouder. I removed leftover docs.

Reply:
added info to index page